Misc/NEWS.d/next/Documentation/2019-10-10-01-00-29.bpo-38409.UQgQk1.rst

```diff
@@ -0,0 +1 @@
+Grammatically corrected help(str.strip) documentation. Patch by Hansraj Das
```

Comment:
Trivial fixes like these don't need a news blurb. I'm gonna apply to the `skip-news` label if you wanna take it out.

Reply:
Deleted news blurb file. Thank you!